### ISSUE TITLE
Add Ancient Greek braille table

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5393]
**Reported by zstanecic on 2015-09-24 20:22**
Hi to the nvda developers from me as the nvda translator.
The following tables are missing from nvda.
gr-bb.ctb, ancient greek (helpful for students who read biblical greek),
hy.ctb, armenian,
is.ctb, icelandic 8 dots
lt.ctb, lithuanian,
pl-pl-comp8.ctb, polish, 8dots table. the latest one
i hope that this can be added in the new master snapshot and that it can be ready for the new version.

**Comment by KayAnderson** [comment]
I will vouch for the correctness of the Ancient Greek table.

**Comment by jcsteh** [comment]
Okay. I'm narrowing the scope of this issue to just Ancient Greek. The others can be addressed separately if others can vouch for them.

Based on a bit of quick research, it looks like Ancient Greek is not really the correct name for this. Would [Koine Greek](https://en.wikipedia.org/wiki/Koine_Greek) be the more correct name?

**Comment by KayAnderson** [comment]
I think they are the same, but Koine is the more academic name. Thank you
for your help with this.

On Monday, February 8, 2016, James Teh <notifications@github.com> wrote:

> Okay. I'm narrowing the scope of this issue to just Ancient Greek. The
> others can be addressed separately if others can vouch for them.
>
> Based on a bit of quick research, it looks like Ancient Greek is not
> really the correct name for this. Would Koine Greek
> <https://en.wikipedia.org/wiki/Koine_Greek> be the more correct name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nvaccess/nvda/issues/5393#issuecomment-181227035>.
>


-- 
http://www.bitingoffmore.wordpress.com

Jesus Christ is love in the flesh.

**Comment by nvaccessAuto** [comment]
Incubated in 8c05cf436fb74fe51741c3ffe721e5bd191e4418.
